easysocial/easysocial/album.php

```diff
@@ -228,16 +217,15 @@ public function create_album()
 		$title	=	$app->input->get('title', 0, 'USER');
```

Comment:
@Pratikshakhandagale - there nothing like user filter in jinput. refer this link - https://docs.joomla.org/Retrieving_request_data_using_JInput

Reply:
This is still not resolved.